### ISSUE TITLE
Add .NET Standard 2.0 support to Microsoft.AspNetCore.WebUtilities

### ISSUE THREAD

**Issue by twsouthwick** [dotnet/aspnetcore#50236]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

During the 2.x timeframe,  `Microsoft.AspNetCore.WebUtilities` was available as a .NET Standard 2.0 NuGet package. It was then moved into the shared framework. However, it is now a package again (as of https://github.com/dotnet/aspnetcore/pull/48265), but only supports .NET 8. People had taken a reliance on it as a package in shared libraries during the migration process but are not able to anymore (they couldn't technically after 2.x, but they just kept the old one it seems).

### Describe the solution you'd like

`Microsoft.AspNetCore.WebUtilities` and `Microsoft.Net.Http.Headers` should support .NET Standard 2.0.

**Comment by BrennanConroy** [comment]
@mkArtakMSFT it looks like your team is doing work with these packages for blazor scenarios. Can you evaluate this issue?

**Comment by BrennanConroy** [comment]
Moving to blazor so the team sees this in triage.

**Comment by twsouthwick** [comment]
> I think if we keep net8.0 and just add netstandard2.0 it will work?

Yup that would help those who have been using the 2.1 version and aren't able to upgrade to the latest 8.0 version.

**Comment by mkArtakMSFT** [comment]
@twsouthwick would you mind sending us a PR with the above proposal ?

**Comment by oatkins** [comment]
> > I think if we keep net8.0 and just add netstandard2.0 it will work?
> 
> Yup that would help those who have been using the 2.1 version and aren't able to upgrade to the latest 8.0 version.

I came to this hoping there would be a 2.0 version... netstandard 2.1 doesn't work for those of us still supporting .NET Framework projects.

Very happy to see the willingness to add this back. Thank you!

**Comment by javiercn** [comment]
Untagging blazor since this is not Blazor specific

**Comment by JobaDiniz** [comment]
Such a lovely and well-written utility to deal with _query strings_ that it should be in a "core" dotnet lib, not part of aspnetcore, since we deal with query strings in other ways, like building urls to `HttpClient` requests...

**Comment by bwilliams1** [comment]
> Such a lovely and well-written utility to deal with _query strings_ that it should be in a "core" dotnet lib, not part of aspnetcore, since we deal with query strings in other ways, like building urls to `HttpClient` requests...

exactly. that is how we use it.
